src/HealthcareApis/HealthcareApis.Test/ScenarioTests/Common.ps1

```diff
@@ -0,0 +1,67 @@
+﻿# ----------------------------------------------------------------------------------
+#
+# Copyright Microsoft Corporation
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+# http://www.apache.org/licenses/LICENSE-2.0
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ----------------------------------------------------------------------------------
+
+<#
+.SYNOPSIS
+Gets valid resource group name
+#>
+function Get-ResourceGroupName
+{
+    return "pstestrg9090"
```

Comment:
These values should not be hard-coded. We have common means to get randomized names for running tests.